lib/controller/StreeplijstController.php

```diff
@@ -0,0 +1,215 @@
+<?php
+
+namespace CsrDelft\controller;
+
+use CsrDelft\common\Annotation\Auth;
+use CsrDelft\common\CsrToegangException;
+use CsrDelft\entity\Streeplijst;
+use CsrDelft\model\entity\LidStatus;
+use CsrDelft\repository\groepen\LichtingenRepository;
+use CsrDelft\repository\groepen\VerticalenRepository;
+use CsrDelft\repository\ProfielRepository;
+use CsrDelft\repository\StreeplijstRepository;
+use CsrDelft\view\JsonResponse;
+use CsrDelft\view\streeplijst\StreeplijstForm;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+use \CsrDelft\view\ToHtmlResponse;
+
+
+/**
+ * StreeplijstController.class.php
+ *
+ * @author J. de Jong
+ *
+ * Controller voor streeplijst generator
+ */
+class StreeplijstController extends AbstractController
+{
+	/**
+	 * @var StreeplijstRepository
+	 */
+	private $streeplijstRepository;
+	/**
+	 * @var ProfielRepository
+	 */
+	private $profielRepository;
+	/**
+	 * @var VerticalenRepository
+	 */
+	private $verticalenRepository;
+
+	public function __construct(
+		StreeplijstRepository $streeplijstRepository, ProfielRepository $profielRepository, VerticalenRepository $verticalenRepository)
+	{
+		$this->streeplijstRepository = $streeplijstRepository;
+		$this->profielRepository = $profielRepository;
+		$this->verticalenRepository = $verticalenRepository;
+	}
+
+	/**
+	 * @return \CsrDelft\view\renderer\TemplateView
+	 * @Route("/streeplijst", methods={"GET"})
+	 * @Auth(P_LOGGED_IN)
+	 */
+	public function overzicht()
+	{
+		return view('streeplijst.overzicht', [
+			'streeplijstoverzicht' => $this->streeplijstRepository->getAlleStreeplijsten(),
+			'streeplijstformulier' => new StreeplijstForm(new Streeplijst()),
+			'huidigestreeplijst' => new Streeplijst(),
+			'verticalen' => $this->verticalenRepository->findAll(),
+			'jongstelidjaar' => LichtingenRepository::getJongsteLidjaar(),
+
+		]);
+
+	}
+
+	/**
+	 * @return \Symfony\Component\HttpFoundation\RedirectResponse
+	 * @Route("/streeplijst/aanmaken", methods={"GET", "POST"})
+	 * @Auth(P_LOGGED_IN)
+	 */
+	public function aanmaken()
+	{
+		$nieuwelijst = $this->streeplijstRepository->nieuw($_GET["naam_streeplijst"], $_GET["leden_streeplijst"], $_GET["inhoud_streeplijst"]);
```

Comment:
Gebruik `Request` met `$request->query->get` zoals je in de `selectie` methode doet.